src/state/opamSysPoll.ml

```diff
@@ -149,7 +149,10 @@ let variables =
   List.map
     (fun (n, v) ->
        OpamVariable.of_string n,
-       lazy (Lazy.force v >>| fun v -> OpamTypes.S v))
+       lazy (try Lazy.force v >>| fun v -> OpamTypes.S v
+             (* We need to catch the assert failure on
+                [OpamSys.with_process_in] for windows *)
```

Comment:
where does this assert come from?

Reply:
https://github.com/ocaml/opam/blob/35c22705f2d7b7121df0cae7ca3662fb3a3e6874/src/core/opamStd.ml#L768-L770